lamindb/base/types.py

```diff
@@ -51,6 +51,7 @@
     "datetime",  # datetime
     "dict",  # dictionary
     "object",  # this is a pandas input dtype, we're only using it for complicated types, not for strings
+    "path",  # path
```

Comment:
Can you also add documentation, please?

Given this is absent in pandera & pandas and only present in pydantic with a different convention, you have to explain how it's exactly defined in LaminDB.

<img width="784" alt="image" src="https://github.com/user-attachments/assets/5792bd28-ee76-42a5-a5d8-fef8c19ca61f" />

You might need to re-organize the table a bit.